docs/references/http-websocket-apis/public-api-methods/clio-methods/server_info-clio.md

```diff
@@ -619,6 +615,17 @@ The `info` object may have some arrangement of the following fields:
 | `validated`                         | Boolean         |  When true, this indicates that the response uses a ledger version that has been validated by consensus. In Clio, this is always true as Clio stores and returns validated ledger data. If a request was forwarded to `rippled` and the server returns current data, a missing or false value indicates that this ledger's data is not final. |
 | `status`                            | String          |  Returns the status of the API request: `success` when the request completes successfully. |
 
+### API Stats Objects
+
+An API Stats object provides key metrics for every API call handled by the Clio server since startup. It includes the following fields:
+
+| `Field`       | Type   | Description                                                                                                  |
+| :------------ | :----- | :----------------------------------------------------------------------------------------------------------- |
+| `started`     | Number | Number of API calls of this type that the Clio server has started processing since startup.                  |
```

Comment:
The only issue I have with this is the -ed endings. `start`, `finish`, `error`, `forward` would be fine. I don't remember seeing past tense fields like this in the past, and while they make sense in context, they read weird to me.